src/app/features/psbt-signer/components/psbt-decoded-request/psbt-unsigned-input-list/components/psbt-unsigned-input-item.tsx

```diff
@@ -23,14 +23,14 @@ export function PsbtUnsignedInputItem({
   const isInputCurrentAddress =
     utxo.scriptpubkey_address === addressNativeSegwit ||
     utxo.scriptpubkey_address === addressTaproot;
-  const inputValue = satToBtc(utxo.value).toString();
+  const inputValueAsMoney = createMoney(utxo.value, 'BTC');
 
   return (
     <PsbtDecodedNodeLayout
       hoverLabel={utxo.scriptpubkey_address}
       subtitle={truncateMiddle(utxo.scriptpubkey_address)}
-      subValue={i18nFormatCurrency(calculateBitcoinFiatValue(inputValue))}
-      value={`${isInputCurrentAddress ? '-' : '+'}${inputValue}`}
+      subValue={i18nFormatCurrency(calculateBitcoinFiatValue(inputValueAsMoney))}
+      value={`${isInputCurrentAddress ? '-' : '+'}${inputValueAsMoney.amount.toString()}`}
```

Comment:
we also have the `formatMoney` fn, don't we want this here?

Reply:
We don't want the symbol here.

Reply:
I could create a new one that doesn't included the symbol?